app/src/main/java/com/mymeetings/android/view/viewModels/CalendarEventsViewModel.kt

```diff
@@ -0,0 +1,69 @@
+package com.mymeetings.android.view.viewModels
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModel
+import com.mymeetings.android.model.CalendarEvent
+import com.mymeetings.android.model.managers.CalendarEventAlertManager
+import com.mymeetings.android.model.managers.CalendarEventsSyncManager
+import com.mymeetings.android.utils.ClockUtils
+import com.mymeetings.android.view.uiModels.CalendarEventAlertUIModel
+import kotlinx.coroutines.*
+
+class CalendarEventsViewModel(
```

Comment:
This class has been moved and modified in the same commit which makes it harder to track changes. Please avoid this from the next commits.